### PR TITLE
Add nav logo, hamburger menu, solutions page route, and footer scroll…

### PR DESCRIPTION
**Pull request by r1g023**
## Jira Issue 47 
[View Jira Issue](https://kseniiariabova.atlassian.net/jira/software/projects/BEE/boards/6?selectedIssue=BEE-47)

- Made sure to pull the latest changes from the Dev branch before starting work on my updates.

### Header Updates
- Imported the Bee logo from Sharaf's Figma design and placed it at the top left corner of the `Header.jsx` component. Spacing and padding adjustments may be needed later; the primary focus was on functionality.

### Nav Updates
- Implemented a responsive design for the `Nav.jsx` component based on Sharaf's recommendation:
  - Nav links are hidden on mobile-sized browsers.
  - A hamburger menu appears in the top right corner of the header on mobile devices.
  - Clicking the hamburger menu displays the Nav menu in a stacked layout below the menu icon and the Bee logo.
  - The hamburger menu toggles between an 'X' icon for closing and the hamburger icon for opening.
  - Used FontAwesome icons for the menu since a Figma reference was not available. The menu design can be updated later based on additional feedback.
  - Added gray borders below each Nav link for a button-like appearance as a temporary measure. Further UX/UI design input from @sharafcs50  may be needed.

### Nav - Footer Updates
- Added an ID to the `footer.jsx` component and linked it in the 'About Us' section of the Nav. Clicking 'About Us' scrolls/jumps to the footer on both mobile and desktop screens.
- Note: No smooth transition was implemented yet. Tailwind CSS has smooth scroll features, or third-party React packages could be used for this in the future. Clicking on the About Us link currently takes the user to the footer section.

### New Solutions Route
- Added `Solutions.jsx` component within the `Components > UI` folder.
- Added a 'Solutions' link to the `Nav.jsx` component.
- Clicking the 'Solutions' link routes to `/solutions`, where the `Solutions.jsx` component is imported within `Routes > AppRoutes.jsx`.
- Implemented as directed by our Scrum Master @KseniiaRiabova 

---

## QA

1. **Refactoring Header Components:** 
   - There are multiple Header components in the project, including Nav. Consider refactoring them into a single folder/section for better organization. This can be addressed in a future task.

2. **Mobile Figma Design Feedback:**
   - Lacking mobile Figma designs for reference. Please provide feedback or suggestions for adjustments, or address this in a future task. @sharafcs50

---

## Attachments

- **New Header Bee Logo**
  Nav_Bee_Logo_Update

- **Mobile Hamburger Nav Menu**
  Mobile_hidden_Menu_update

- **Mobile Open Nav Menu**
  Mobile_open_Menu_update

- **New Solutions Route Page**
  New_Solutions_Route

### CODE REVIEW

**Review by NishaVijai — APPROVED** [review]
Thank you for an awesome work @r1g023  :-)

File: client/src/components/UI/Nav.jsx
Comment (NishaVijai) [inline client/src/components/UI/Nav.jsx:1]: I am making some changes on this file in order to add burger menu :-)

**Review by KseniiaRiabova — APPROVED** [review]
Rigo, this is the best PR ever! Awesome Job, I am so proud of your progress!

**Review by PongRVA — APPROVED** [review]
Looks awesome, @r1g023

**Review by Valeriusdev — APPROVED** [review]

**Review by ss-deep — APPROVED** [review]
Looks great!

**Pull request merged**